webview_panels/src/lib/altimate/altimate-components.d.ts

```
@@ -1,150 +1,596 @@
+
+export declare class CLL {
+    static isCancelled: boolean;
+    static inProgress: boolean;
+    static linkCount: number;
+    static onCancel(): void;
+    static cancel(): void;
+    static start(): void;
+    static end(): void;
+    static addLinks(n: number): void;
+    static showCllInProgressMsg(): void;
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)